dist/samples/store-locator/index.js

```diff
@@ -10,10 +12,21 @@ const stores = [];
 // Initialize and add the map
 function initMap() {
   distanceMatrixService = new google.maps.DistanceMatrixService();
+  originalMapTypeId = google.maps.MapTypeId.ROADMAP;
+  detailMapTypeId = google.maps.MapTypeId.HYBRID;
   map = new google.maps.Map(document.getElementById("map"), {
     center: { lat: 39.79, lng: -104.98 },
+    mapTypeId: originalMapTypeId,
     zoom: 10,
   });
+  map.addListener("zoom_changed", () => {
+    const newZoom = map.getZoom();
+
+    if (newZoom < 19) {
+      map.setTilt(0);
+      map.setMapTypeId(originalMapTypeId);
+    }
+  });
   new mdc.textField.MDCTextField(document.querySelector(".mdc-text-field"));
```

Comment:
we can now import instead of using the cdn version, but that is probably out of scope.

Reply:
ack